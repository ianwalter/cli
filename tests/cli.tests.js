const { test } = require('@ianwalter/bff')
const execa = require('execa')

test('cli', async ({ expect }) => {
  const { stdout } = await execa('./tests/helpers/exampleCli.js', ['-c', '4'])
  expect(stdout).toMatchSnapshot()
})
