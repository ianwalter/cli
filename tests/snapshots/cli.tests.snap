// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cli 1`] = `"{ level: 'debug', concurrency: 4, _: [], c: 4 }"`;
